src/Workspaces/Core/Desktop/Workspace/SQLite/SQLitePersistentStorage.cs

```diff
@@ -243,6 +243,11 @@ public override void Initialize(Solution solution)
             {
                 var connection = pooledConnection.Connection;
 
+                // Configure journal and locking mode
+                connection.ExecuteCommand("pragma locking_mode=exclusive", throwOnError: false);
+                connection.ExecuteCommand("pragma journal_mode=wal", throwOnError: false);
+                connection.ExecuteCommand("pragma synchronous=normal", throwOnError: false);
```

Comment:
We should document *in code* all the decision making and data that is in this PR.  It will be very useful for future readers to understand what's going on here, and why it's ok.  For example, why "normal" mode is an acceptable default for us given the needs of our cache.

Reply:
Agreed.